packages/eas-cli/src/utils/expodash/filter.ts

```diff
@@ -0,0 +1,4 @@
+/** Filter the array from `null` and `undefined` values, returning a type-safe array. */
+export default function filter<T>(list: (T | undefined | null)[]): T[] {
+  return list.filter<T>((i): i is T => i !== undefined && i !== null);
+}
```

Comment:
optional: alternatively you could use predicates in combination with the builtin `filter` method. From www:
```typescript
export function truthy<TValue>(value: TValue | null | undefined): value is TValue {
  return !!value;
}

const b = a.filter(truthy);
```

Reply:
I like that pattern, but I'm unsure how it would fit in the `expodash` functions. We can just drop them in there and go from there? 😄 (cc @dsokal)

Reply:
I think it fits in expodash well, so go ahead :)

Reply:
Done!

Reply:
Done!